### ISSUE TITLE
Usage of "aria-detailed-id" Also in dropdowns/textarea/etc

### ISSUE THREAD

**Issue by EdsonT-Systems** [telekom/scale#2356]
**Scale Version**
Latest

**Framework and version**
Angular latest with  auto-generated scale wrapper packages

**Current Behavior**
The "aria-details-id" property doesn’t work without the "helper-text" property(only works in text-field)

**Expected Behavior** **Code Reproduction**
The expected behavior is for the property to work without relying on others.

Last time, an adaptation was requested to use the aria-details-id property without the use of the helper-text. It was adapted for the text-field but not for the rest of the fields. Would it be possible to adapt it to them as well? Thanks 

@tshimber @acstll @felix-ico @amir-ba
